features/src/icc/.bashrc

```diff
@@ -0,0 +1,2 @@
+export ICC_VERSION="${ICC_VERSION}";
+export PATH="$PATH";
```

Comment:
I've started doing this pattern to avoid modifying PATH if it already has what we want:

```suggestion
if [ -n "${PATH##*"/opt/intel/oneapi/compiler/${ICC_VERSION}/linux/bin"*}" ]; then
    export PATH="$PATH:/opt/intel/oneapi/compiler/${ICC_VERSION}/linux/bin";
fi
```